.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -9,22 +9,15 @@ assignees: ''
 
 ## Pre-release
 
-- [ ] Add build targets for the new release on [Docker Hub]
-  - All versions:
-    - [cilium](https://hub.docker.com/repository/docker/cilium/cilium/builds/edit)
-    - [operator](https://hub.docker.com/repository/docker/cilium/operator/builds/edit)
-    - [docker-plugin](https://hub.docker.com/repository/docker/cilium/docker-plugin/builds/edit)
-  - Cilium v1.8 or later:
-    - [operator-generic](https://hub.docker.com/repository/docker/cilium/operator-generic/builds/edit)
-    - [operator-aws](https://hub.docker.com/repository/docker/cilium/operator-aws/builds/edit)
-    - [operator-azure](https://hub.docker.com/repository/docker/cilium/operator-azure/builds/edit)
-    - [hubble-relay](https://hub.docker.com/repository/docker/cilium/hubble-relay/builds/edit)
-  - Cilium v1.9 or later:
-    - [clustermesh-apiserver](https://hub.docker.com/repository/docker/cilium/clustermesh-apiserver/builds/edit)
 - [ ] Check that there are no [release blockers] for the targeted release version
 - [ ] Ensure that outstanding [backport PRs] are merged
 - [ ] Consider building new [cilium-runtime images] and bumping the base image
-      versions on this branch
+      versions on this branch:
+  - Cilium v1.10 or later:
+     Simply modify the runtime Dockerfile and add a `ENV` variable to force a rebuild.
+     [Instructions](https://docs.cilium.io/en/latest/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images)
```

Comment:
These instructions are the same as below, just with a different URL right? I don't see anything about an `ENV` variable.

Reply:
It depends on https://github.com/cilium/cilium/pull/15023